sql-cli/sql_cli/project.py

```diff
@@ -0,0 +1,57 @@
+import os
+import shutil
+from pathlib import Path
+from typing import Optional
+
+from sql_cli.configuration import DEFAULT_ENVIRONMENT, Config
+
+BASE_SOURCE_DIR = Path(os.path.realpath(__file__)).parent.parent / "include/base/"
+
+
+class Project:
+    """
+    SQL CLI Project.
+    """
+
+    def __init__(
+        self, directory: Path, airflow_dags_folder: Optional[Path] = None, airflow_home: Optional[Path] = None
+    ) -> None:
+        self.directory = directory
+        self._airflow_dags_folder = airflow_dags_folder
+        self._airflow_home = airflow_home
+
+    def _delete_temporary_files(self) -> None:
+        """
+        Delete files which should not be part of the user project directory structure.
+        An example are the .gitkeep files, which were added just so the default folder structure
+        can be added to Git (by default empty directories are not versioned).
+        """
+        gitkeep_files = Path(self.directory).rglob(".gitkeep")
+        for file_ in gitkeep_files:
+            file_.unlink()
+
+    def _update_config(self, airflow_home: Optional[Path], airflow_dags_folder: Optional[Path]) -> None:
```

Comment:
Why not use the `self._airflow_dags_folder` and `self._airflow_home` ?

Reply:
We could, but here I prefer to be explicit about which config properties may be changed.